### DISCUSSION TITLE
Forget password

### DISCUSSION THREAD

**Discussion by al3rabi1** [community/community#23106] [New to GitHub]
<details>
<summary>
Summary</summary>
<p>This text will be hidden</p>
</details>

**Comment by Akash1134** [comment, marked as answer]
<p>:wave: Hi <a class="mention" href="/al3rabi1">@al3rabi1</a> , Welcome to the community! We are happy you are here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction along with posting your question(s) in the applicable category. I’ve gone ahead and moved it for you. Good luck!</p>

**Comment by Musa0813** [comment]
I forget my github account passowrd
